### ISSUE TITLE
Error message flooding with `[BUG] Itag <<XXX>> is not recognized!`

### ISSUE THREAD

**Issue by TheChymera** [trizen/pipe-viewer#184]
Whenever I play a video, as far as I can tell *any* video, my terminal gets flooded with messages like this.

Any ideas:
* Whether this is an issue which it would be worth addressing for any other reasons than flooding my terminal?
* If not, whether I can more easily just suppress these messages somehow?

```
=>> Select one or more videos to play (:h for help)
> 1
[BUG] Itag <<229>> is not recognized!
{
  itag => 229,
  type => "video/mp4",
  url  => "https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1695246789/ei/ZBULZaPeOt6t_9EP8-ypwAk/ip/2601:19e:8280:213:1183:f75d:9dbf:f8f5/id/5401fe8b1745585b/itag/229/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D16597663%3Bdur%3D1200.899%3Bgir%3Dyes%3Bitag%3D133%3Blmt%3D1681747256304447/hls_chunk_host/rr10---sn-jvhj5nu-cvnee.googlevideo.com/mh/CM/mm/31,29/mn/sn-jvhj5nu-cvnee,sn-ab5l6nrz/ms/au,rdu/mv/m/mvi/10/pl/40/initcwndbps/1082500/vprv/1/playlist_type/DVR/dover/13/txp/453D434/mt/1695224720/fvip/3/short_key/1/keepalive/yes/fexp/24007246/beids/24350018/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,vprv,playlist_type/sig/AOq0QJ8wRAIgKY1VHitSmrkLR1G39yvFtIpFrC7L2d6Tc1hqQC1BOH4CIE4rIxIe9QQWAWnqWvvj4reVil-Mji5VP_7y4V3Qs4Vc/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AG3C_xAwRQIhAL2Yq4cmdLgf2xdP2u97MN4gf6RtD6zMmmk8EQ6v6FGMAiBH41JpJP8d5VkMiBDumAWLryeitEnRJ1UYFPlcvW9Quw%3D%3D/playlist/index.m3u8",
}
[BUG] Itag <<230>> is not recognized!
{
  itag => 230,
  type => "video/mp4",
  url  => "https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1695246789/ei/ZBULZaPeOt6t_9EP8-ypwAk/ip/2601:19e:8280:213:1183:f75d:9dbf:f8f5/id/5401fe8b1745585b/itag/230/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D30814206%3Bdur%3D1200.899%3Bgir%3Dyes%3Bitag%3D134%3Blmt%3D1681747210017989/hls_chunk_host/rr10---sn-jvhj5nu-cvnee.googlevideo.com/mh/CM/mm/31,29/mn/sn-jvhj5nu-cvnee,sn-ab5l6nrz/ms/au,rdu/mv/m/mvi/10/pl/40/initcwndbps/1082500/vprv/1/playlist_type/DVR/dover/13/txp/453D434/mt/1695224720/fvip/3/short_key/1/keepalive/yes/fexp/24007246/beids/24350018/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,vprv,playlist_type/sig/AOq0QJ8wRgIhAJII2N7CZa4k4ty0LupPyAnf_2boYoLskH-LCxJudEiEAiEA_J9wOZy9GLpAp5Ly5ezUy9q-vwu2jqesHQ6n-f-L-dA%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AG3C_xAwRgIhAM5MIwteMli_fX4B86PmEUSp94K1PO_IfJ0CNdJKZLDSAiEArG110n9VfGG5uuUlyp4uQiA3TGeaiWxUvvRaieSxBYI%3D/playlist/index.m3u8",
}
[BUG] Itag <<231>> is not recognized!
{
  itag => 231,
  type => "video/mp4",
  url  => "https://manifest.googlevideo.com/api/manifest/hls_playlist/expire/1695246789/ei/ZBULZaPeOt6t_9EP8-ypwAk/ip/2601:19e:8280:213:1183:f75d:9dbf:f8f5/id/5401fe8b1745585b/itag/231/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/sgovp/clen%3D47586118%3Bdur%3D1200.899%3Bgir%3Dyes%3Bitag%3D135%3Blmt%3D1681747327552404/hls_chunk_host/rr10---sn-jvhj5nu-cvnee.googlevideo.com/mh/CM/mm/31,29/mn/sn-jvhj5nu-cvnee,sn-ab5l6nrz/ms/au,rdu/mv/m/mvi/10/pl/40/initcwndbps/1082500/vprv/1/playlist_type/DVR/dover/13/txp/453D434/mt/1695224720/fvip/3/short_key/1/keepalive/yes/fexp/24007246/beids/24350018/sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,sgovp,vprv,playlist_type/sig/AOq0QJ8wRAIgBfp93Rfhx3uiqP78TllOr0RTBBR1IRFiYuLN6rWL4R8CIDhIq5VKHEwJKfIT1MYetHpzJpxXH8DkkSDL9PZdGIeI/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,pl,initcwndbps/lsig/AG3C_xAwRgIhAICEHeGXTPXGpb3ue3CVJ41lhcjSnJYSX6bwYKvxOqcTAiEA1kaApluoRiH2YmWGBbK-jTozRiziQawMBcOD2RWPWPM%3D/playlist/index.m3u8",
}
```

**Comment by trizen** [comment]
I can replicate the issue with the `force_fallback` config-option enabled.

The warnings can be silenced by adding the corresponding itags in the [itags file](https://github.com/trizen/pipe-viewer/blob/main/lib/WWW/PipeViewer/Itags.pm), or by disabling the `force_fallback` option.

**Comment by trizen** [comment]
The issue should be fixed in version 0.4.9. Make sure to have the latest version installed by checking:

```
pipe-viewer --version
```
